execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -104,13 +110,38 @@ pub struct WasmTestBuilder<S> {
     standard_payment_hash: Option<ContractHash>,
     /// Auction contract key
     auction_contract_hash: Option<ContractHash>,
+    /// Scratch global state used for in-memory execution and commit optimization.
+    scratch_engine_state: Option<EngineState<ScratchGlobalState>>,
 }
 
 impl<S> WasmTestBuilder<S> {
     fn initialize_logging() {
         let log_settings = Settings::new(LevelFilter::Error).with_style(Style::HumanReadable);
         let _ = logging::initialize(log_settings);
     }
+
+    /// Returns execution results.
+    pub fn exec_results(&self) -> Vec<Vec<Rc<ExecutionResult>>> {
+        self.exec_results
+            .iter()
+            .map(|entry| {
+                entry
+                    .iter()
+                    .cloned()
+                    .map(|(_deploy_hash, result)| result)
+                    .collect()
+            })
+            .collect()
+    }
+
+    /// Returns transforms.
+    pub fn transforms(&self) -> Vec<AdditiveMap<Key, Transform>> {
```

Comment:
I would return `DeployHash`-es as well, so that the caller can decide whether to use it/correlate with an order of the submitted txns.

Reply:
Refactored this to return the deploys, however the above `exec_results` method is not needed anymore.